### DISCUSSION TITLE
Where are stored qBittorrent configuration files and logs ?

### DISCUSSION THREAD

**Discussion by vincegre** [swizzin/swizzin#1179] [Q&A + Help]
Hi

I have setup a seedbox using swizzin and it works like a charm, thanks for the great work. Unhappy I have a little issue with qBittorrent and so I would need access at configuration files and logs of qBittorrent ! Where are they stored ? as I can't find them in usual places :(

Thanks

Vincèn

**Comment by siedenburg2** [comment]
Did you check the files in /home/username/.config/qBittorrent? If you open the qBittorrent.conf you can see the path for logs
